### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Kateryna-Bordonos17**

### CODE REVIEW

**Review by OlehShumov — APPROVED** [review]
None -> Any

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,50 @@
...
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self, distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int) -> None:
```
Comment (LiudmylaKulinchenko) [inline app/main.py]: fix formatting here

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,51 @@
...
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int) -> None:
```
Comment (LiudmylaKulinchenko) [inline app/main.py]: fix formatting

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

**Review by LiudmylaKulinchenko — APPROVED** [review]
Well done! 🎈😸

**Pull request closed without merging**